.buildkite/pipelines/pull_request/defend_workflows.yml

```diff
@@ -22,3 +22,28 @@ steps:
       automatic:
         - exit_status: '*'
           limit: 1
+
+  - command: .buildkite/scripts/steps/functional/defend_workflows_serverless.sh
```

Comment:
Do we only want this to run conditionally on pull requests?

on-merge at https://github.com/elastic/kibana/blob/main/.buildkite/pipelines/on_merge.yml#L82
all pull requests at https://github.com/elastic/kibana/blob/main/.buildkite/pipelines/pull_request/base.yml#L71

Reply:
thanks for the question! I've added it to our [tracking issue](https://github.com/elastic/security-team/issues/7610) for serverless tests, to make a team decision - until then, for this PR, I'd keep it this way to merge this as soon as possible.